infra.bs

```diff
@@ -608,58 +608,58 @@ U+007A (z), inclusive.
 
 <h3 id=strings>Strings</h3>
 
-<p>A <dfn export>JavaScript string</dfn> is a sequence of unsigned 16-bit integers, also known as
-<dfn export lt="code unit">code units</dfn>.
+<p>A <dfn export>string</dfn> is a sequence of unsigned 16-bit integers, also known as
+<dfn export lt="code unit">code units</dfn>. A <a>string</a> is also known as a
+<dfn export>JavaScript string</dfn>. <a>Strings</a> are denoted by double quotes and monospace font.
```

Comment:
I'm happy with this, but want to mention a couple alternatives/additions:

- State situations in which one might prefer to use "JavaScript string" instead of just "string", e.g. "A string is also known as a JavaScript string; typically this is used to emphasize the nature of the string as a collection of code units, as opposed to the below-discussed interpretation as containing code points."

- Use `<dfn export lt="string,JavaScript string">string</dfn>` and remove the `<dfn>` for JavaScript string, either converting it to a link or just leaving it without markup.

Reply:
I didn't go with "typically" since I'm not really sure what will be the dominant pattern long term.